set.mm

```diff
@@ -531681,10 +531688,81 @@ sides of the biconditional correlate (they are the same), if they exist at
     $( Alternate definition of restricted universal quantification when ` x `
        and ` A ` are disjoint.  (Contributed by Wolf Lammen, 23-May-2023.) $)
     wl-dfralv $p |- ( A. x wl-in A ph <-> A. x ( x e. A -> ph ) ) $=
-      ( vy wwl-in cv wcel weq wal df-wl-ral 19.21v albii wl-nfcrallem 3bitr2i
+      ( vy wl-ral cv wcel weq wal df-wl-ral 19.21v albii wl-nfcrallem 3bitr2i
       wi ) ABCEDFCGZBDHAOZBIOZDIPQOBIZDIBFCGAOBIABDCJSRDPQBKLABDCMN $.
   $}
 
+  ${
+    $d x y $.  $d A y $.  $d ph y $.
+    $( An alternate definition of restricted universal quantification
+       ( ~ df-wl-ral ) using substitution.  (Contributed by Wolf Lammen,
+       25-May-2023.) $)
+    wl-dfral2 $p |- ( A. x wl-in A ph <-> A. y ( y e. A -> [ y / x ] ph ) ) $=
+      ( wl-ral cv wcel weq wi wal wsb df-wl-ral sb6 imbi2i albii bitr4i ) ABDEC
+      FDGZBCHAIBJZIZCJQABCKZIZCJABCDLUASCTRQABCMNOP $.
+  $}
```

Comment:
There is already a `dfral2` in the main part of set.mm. it might conflict if we want to merge those changes into the main part.

Reply:
Same remark for `~dfrex2`